addon/mixins/sortable-item.js

```diff
@@ -308,14 +308,18 @@ export default Mixin.create({
     const groupDirection = this.get('group.direction');
     let dragOrigin;
     let elementOrigin;
+    let scrollOrigin;
 
     if (groupDirection === 'x') {
       dragOrigin = getX(startEvent);
       elementOrigin = this.get('x');
+      scrollOrigin = $(this.element).offset().left;
 
       return event => {
         let dx = getX(event) - dragOrigin;
-        let x = elementOrigin + dx;
+        let elementX = this.get('x') - this.element.offsetLeft;
```

Comment:
Gotcha. I don’t know if there’s a better approach either. Maybe let’s just add some comments to illuminate?